src/index.html

```diff
@@ -6,16 +6,24 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>JSHeroes Bootcamp</title>
     <link rel="stylesheet" href="style.css" />
+    <script src='script.js' defer></script>
   </head>
   <body>
     <header>
-      <h1>Welcome to the JSHeroes Bootcamp!</h1>
+      <h1>GitHub Explorer</h1>
     </header>
     <main>
-      <ul class="repo-list">
-        <!-- repo list -->
+      <ul class="repo-cards">
+        <!--       Card example     -->
+        <li>
+          <span>test header</span>
```

Comment:
maybe we wanna make this a `<h3>` ?

Reply:
Good idea! Fixed it :)

Reply:
Not sure this is a good idea as headings should be in the correct order, so the semantically correct tag could be h2. However, when using lists, headings are not that relevant, as they don't separate bigger sections of text/content.  I would propose keeping it a <span> with a different class for styling maybe